### ISSUE TITLE
NTR: Tetrahydrobiopterin high in CSF (Lab: cerebrospinal fluid tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8483]
New term request Tetrahydrobiopterin high in CSF (elthat_181122073055): BH4; cerebrospinal fluid; sapropterin

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000822
- label: Elevated CSF tetrahydrobiopterin level
- parents: [Abnormal CSF biopterin concentration (HP:0040207)](https://hpo.jax.org/app/browse/term/HP:0040207)
- pmid: 
- def: Concentration of tetrahydrobiopterin in the cerebrospinal fluid (CSF) above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8483
- synonyms:
